packages/ui/src/components/AppContent/index.module.scss

```diff
@@ -42,7 +42,7 @@ $font-family-small: 'PingFang SC', 'SF Pro Text', 'Siyuan Heiti', 'Roboto';
   --color-layer: linear-gradient(0deg, rgba(202, 190, 255, 14%), rgba(202, 190, 255, 14%)), linear-gradient(0deg, rgba(196, 199, 199, 2%), rgba(196, 199, 199, 2%)), #191c1d;
   --color-error: #dd3730;
   --color-toast: rgba(247, 248, 248, 80%);
-  --color-overlay: rgba(247, 248, 248, 80%);
+  --color-overlay: rgba(247, 248, 248, 16%);
```

Comment:
nit: Why this color will change here, maybe this belongs to another PR.

Reply:
yes, it's a bug found when testing this pr.  Hard to find a proper PR to update this simple issue, so leave it here.